README.md

```diff
@@ -59,11 +65,11 @@ For example `ca mycafile.pem` becomes
 ```
 Footnotes
 -----------
-Please note that OpenVPN used by this project is under GPLv2. 
+Please note that OpenVPN used by this project is under GPLv2+.
 
-If you cannot or do not want to use the Play Store you can [download the apk files directly](http://plai.de/android/).
+If you cannot or do not want to use the Play Store you can [download the APK files directly](http://plai.de/android/).
 
-If you want to donate you can donate to [arne-paypal@rfc2549.org via paypal](https://www.paypal.com/cgi-bin/webscr?hosted_button_id=R2M6ZP9AF25LS&cmd=_s-xclick), or alternatively if you believe in fancy Internet money you can use Bitcoin: 1EVWVqpVQFhoFE6gKaqSkfvSNdmLAjcQ9z 
+If you want to be a hero, donate to [arne-paypal@rfc2549.org via paypal](https://www.paypal.com/cgi-bin/webscr?hosted_button_id=R2M6ZP9AF25LS&cmd=_s-xclick), or alternatively if you believe in fancy Internet money (or not) you can use Bitcoin: 1EVWVqpVQFhoFE6gKaqSkfvSNdmLAjcQ9z 
```

Comment:
again, you change the tone/meaning here.

Reply:
You start to really annoy me with assuming what would be best for me. Have you ever thought that I don't want to entice people to donate and just this neutral tone is exactly what I want?